sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-jedis/src/test/java/com/azure/messaging/eventhubs/checkpointstore/jedis/JedisRedisCheckpointStoreTests.java

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.messaging.eventhubs.checkpointstore.jedis;
+
+import com.azure.core.util.serializer.JsonSerializer;
+import com.azure.core.util.serializer.JsonSerializerProviders;
+import com.azure.messaging.eventhubs.models.Checkpoint;
+import com.azure.messaging.eventhubs.models.PartitionOwnership;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import reactor.test.StepVerifier;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * Unit tests for {@link  JedisRedisCheckpointStore}
+ */
+public class JedisRedisCheckpointStoreTests {
+    private JedisPool jedisPool;
+    private JedisRedisCheckpointStore store;
+    private Jedis jedis;
+    private JsonSerializer jsonSerializer;
+    @BeforeEach
+    public void setup() {
+        jedisPool = mock(JedisPool.class);
+        jedis = mock(Jedis.class);
+        store = new JedisRedisCheckpointStore(jedisPool);
+        jsonSerializer = JsonSerializerProviders.createInstance(true);
+    }
+
+    @Test
+    public void testListCheckpoints() {
+        //arrange
+        Checkpoint checkpoint = new Checkpoint()
+            .setConsumerGroup("consumerGroup")
+            .setEventHubName("eventHubName")
+            .setFullyQualifiedNamespace("fullyQualifiedNamespace")
+            .setPartitionId("one")
+            .setSequenceNumber((long) 1);
+        Set<String> value = new HashSet<>();
+        List<String> list = new ArrayList<>();
+        value.add("fullyQualifiedNamespace/eventHubNamespace/consumerGroup/one");
+        byte[] bytes = jsonSerializer.serializeToBytes(checkpoint);
+        list.add(new String(bytes));
```

Comment:
1. I'd consider specifying the character set. `new String(bytes, StandardCharset.UTF_8)`
2. Since this array has a single item, can we use `Collections.singletonlist` or `Arrays.asList()` to clean up the declarations?

Reply:
Thanks Connie! I didn't know about the concept of singletonList but I have gone ahead and used it now! Thank you :)